Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -119,8 +126,11 @@ public void CLIVersion()
 
             // Act            
             CLIProcessor CLI = new CLIProcessor();
-            CLI.ExecuteArgs(new string[] { "-t" });
-
+           
+            Task.Run(async () =>
+            {
+                await CLI.ExecuteArgs(new string[] { "-t" });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)